aten/src/ATen/native/README.md

```diff
@@ -277,6 +277,18 @@ them the same thing!)
 If two backends have the same dispatch function, you can write `CPU, CUDA: func`
 to reuse the same function name in both cases.
 
+TODO: expand this section.
```

Comment:
(another drive-by comment): I don't know what the Math key is but it looks interesting. Would love to see an explanation of what it's used for here

Reply:
https://fb.quip.com/jiYeAa58PDX0 explains it pretty well. But yea I should improve docs here as well. ;)

Reply:
Yeah, let's publish the design on a GitHub issue so non-FB people can see it.